src/ethereum_test_rpc/types.py

```diff
@@ -136,9 +136,22 @@ def blob_versioned_hashes(self) -> List[Hash]:
         return [Hash(b"\1" + commitment[1:]) for commitment in self.commitments]
 
 
+class BlobAndProof(CamelModel):
+    """Represents a blob and proof structure."""
+
+    blob: Bytes
+    proofs: List[Bytes]
+
+
 class GetPayloadResponse(CamelModel):
     """Represents the response of a get payload request."""
 
     execution_payload: FixtureExecutionPayload
     blobs_bundle: BlobsBundle | None = None
     execution_requests: List[Bytes] | None = None
+
+
+class GetBlobsResponse(CamelModel):
+    """Represents the response of a get blobs request."""
+
+    result: List[BlobAndProof]
```

Comment:
```suggestion
    result: List[BlobAndProof | None]
```
Reading from the "or null in case of any missing blobs" in https://github.com/ethereum/execution-apis/blob/main/src/engine/osaka.md#response-1